projects/vdk-control-cli/.gitlab-ci.yml

```diff
@@ -10,6 +10,7 @@ image: "python:3.9"
   stage: build
   before_script:
     - cd projects/vdk-control-cli
+    - pip install --upgrade pytest pip # TODO: REMOVE THIS AT SOME POINT
```

Comment:
TODO? Is this necessary? as it's installed by build.sh ?

Reply:
I tested and it's not. I had it included at the start due to some issue, but I guess the cause was different.